projects/Mallard/src/screens/settings-screen.tsx

```diff
@@ -127,6 +130,10 @@ const DevZone = withNavigation(({ navigation }: NavigationInjectedProps) => {
     )
 })
 
+const rightChevronIcon = (
+    <RightChevron textColor={useAppAppearance().color} fontSize={22} />
```

Comment:
this looks like a sensible default, i'd put it inside `<RightChevron/>` and that way you can just drop it around

Reply:
Done.